.nais/vars/vars-dev2.yml

```diff
@@ -2,6 +2,7 @@ appName: nav-enonicxp-frontend-dev2
 revalidatorApp: nav-enonicxp-frontend-revalidator-proxy-dev2
 decoratorHost: dekoratoren.ekstern.dev.nav.no
 xpHost: www-q6.nav.no
+dev2Host: www-2.dev.nav.no
```

Comment:
Dette bør nok skrives om litt på en mer generell måte, kanskje til en liste med external hosts for hvert miljø.

Reply:
https://github.com/navikt/nav-enonicxp-frontend/pull/1316